pkg/server/api.go

```diff
@@ -31,8 +27,13 @@ type APIServer struct {
 // that runs the Machine Config Server as a
 // handler.
 func NewAPIServer(a *APIHandler, p int, is bool, c, k string) *APIServer {
+	mux := http.NewServeMux()
+	mux.Handle("/config/", a)
```

Comment:
Why take an `*APIHandler` argument, when the only way to get an `*APIHandler` with `.server` set is to call `NewServerAPIHandler`?  Seems like it would be easier to have `NewAPIServer` take a `Server` argument (although see [here][1] about naming that interface `Server`).

[1]: https://github.com/openshift/machine-config-operator/pull/246#issuecomment-448053607